asset-transfer-basic/chaincode-external/assetTransfer.go

```diff
@@ -0,0 +1,219 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/hyperledger/fabric-chaincode-go/shim"
+	"github.com/hyperledger/fabric-contract-api-go/contractapi"
+)
+
+type ServerConfig struct {
+	CCID    string
+	Address string
+}
+
+// SmartContract provides functions for managing a car
+type SmartContract struct {
+	contractapi.Contract
+}
+
+// Asset describes basic details of what makes up a simple asset
+type Asset struct {
```

Comment:
No need to export

Reply:
`Asset` is being returned from an exported function `ReadAsset`, unexport it will be annoy to be used.